### DISCUSSION TITLE
Is there a clean way to add ores to planets

### DISCUSSION THREAD

**Discussion by Bluedot55** [terrarium-earth/Ad-Astra#226] [Q&A]
For modpacks, is there a way to integrate ores from other mods with the ad astra worlds?

**Comment by AlexNijjar** [comment]
you have to do it in a data pack by overriding a specific biome such as https://github.com/terrarium-earth/Ad-Astra/blob/1.19.3/common/src/main/resources/data/ad_astra/worldgen/biome/lunar_wastelands.json

and then adding additional ores to the features array
